From bfb24a7eb81a8ee274d8b883e48ae6461185ae23 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Wed, 15 Aug 2018 11:02:48 +0800
Subject: [PATCH 31/54] media: intel-ipu4: css: scci_IPU4_master_20180813_1222
 fw release

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
FW_Manifest_ID: 0eb7e4d6711340d4bdecc9ae29956f2ddabac0de
FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id f109844
FW Bundle patch number of this release :  348195
FW tag : scci_IPU4_master_20180813_1222

Change-Id: I2f9f74b2e73e872860c93b19b373771f07d57d34
Depends-on: I738bee24283ad25cdd9f02184ad4a015bd7f4e50
Depends-on: Ibe6e045050335ad9373fd1bbe378d45ae91cc066
Depends-on: I0cb3ab490328d366a28c690448f943b3aad61657
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h        | 2 +-
 .../ipu4/ipu4-css/lib2600/fw_abi_common_types/ia_css_terminal_defs.h | 4 ----
 .../intel/ipu4/ipu4-css/lib2600/isysapi/interface/ia_css_isysapi.h   | 5 -----
 .../media/pci/intel/ipu4/ipu4-css/lib2600/support/assert_support.h   | 3 ---
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h     | 2 +-
 .../lib2600psys/lib/fw_abi_common_types/ia_css_terminal_defs.h       | 4 ----
 .../pci/intel/ipu4/ipu4-css/lib2600psys/lib/support/assert_support.h | 3 ---
 7 files changed, 2 insertions(+), 21 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index ff6c501..42df87e 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180809
+#define IA_CSS_FW_PKG_RELEASE  0x20180813
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/fw_abi_common_types/ia_css_terminal_defs.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/fw_abi_common_types/ia_css_terminal_defs.h
index 3a7b333..dbf1cf9 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/fw_abi_common_types/ia_css_terminal_defs.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/fw_abi_common_types/ia_css_terminal_defs.h
@@ -71,10 +71,6 @@
  * Dimensions of the data objects. Note that a C-style
  * data order is assumed. Data stored by row.
  */
-/* A strange problem with hivecc compiler which is described
- * here https://icggerrit.ir.intel.com/#/c/51630/1 forces this
- * enum to be explicitly initialized for the moment
- */
 typedef enum ia_css_dimension {
 	/**< The number of columns, i.e. the size of the row */
 	IA_CSS_COL_DIMENSION = 0,
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/isysapi/interface/ia_css_isysapi.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/isysapi/interface/ia_css_isysapi.h
index 5f10f72..abbc8b8 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/isysapi/interface/ia_css_isysapi.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/isysapi/interface/ia_css_isysapi.h
@@ -15,11 +15,6 @@
 #ifndef __IA_CSS_ISYSAPI_H
 #define __IA_CSS_ISYSAPI_H
 
-/**
- * errno.h specified error codes to be used
- * URL:http://man7.org/linux/man-pages/man3/errno.3.html>
- */
-
 
 /* The following is needed for the function arguments */
 #include "ia_css_isysapi_types.h"
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/support/assert_support.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/support/assert_support.h
index 28aed19..f904a49 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/support/assert_support.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/support/assert_support.h
@@ -39,9 +39,6 @@
  * should not depend on assert to function (actually the assert
  * could be disabled in a release build) it was decided to
  * disable the assert for KW scans (by defining NDEBUG)
- * see also:
- * http://www.klocwork.com/products/documentation/current/
- * Tuning_C/C%2B%2B_analysis#Assertions
  */
 #define NDEBUG
 #endif /* __KLOCWORK__ */
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index ff6c501..42df87e 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180809
+#define IA_CSS_FW_PKG_RELEASE  0x20180813
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/fw_abi_common_types/ia_css_terminal_defs.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/fw_abi_common_types/ia_css_terminal_defs.h
index 3a7b333..dbf1cf9 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/fw_abi_common_types/ia_css_terminal_defs.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/fw_abi_common_types/ia_css_terminal_defs.h
@@ -71,10 +71,6 @@
  * Dimensions of the data objects. Note that a C-style
  * data order is assumed. Data stored by row.
  */
-/* A strange problem with hivecc compiler which is described
- * here https://icggerrit.ir.intel.com/#/c/51630/1 forces this
- * enum to be explicitly initialized for the moment
- */
 typedef enum ia_css_dimension {
 	/**< The number of columns, i.e. the size of the row */
 	IA_CSS_COL_DIMENSION = 0,
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/support/assert_support.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/support/assert_support.h
index 28aed19..f904a49 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/support/assert_support.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/support/assert_support.h
@@ -39,9 +39,6 @@
  * should not depend on assert to function (actually the assert
  * could be disabled in a release build) it was decided to
  * disable the assert for KW scans (by defining NDEBUG)
- * see also:
- * http://www.klocwork.com/products/documentation/current/
- * Tuning_C/C%2B%2B_analysis#Assertions
  */
 #define NDEBUG
 #endif /* __KLOCWORK__ */
-- 
1.9.1

